tests/integration/test_catboost_evaluate/test.py

```diff
@@ -65,41 +74,61 @@ def testModelPathIsNotAConstString(ch_cluster):
     if instance.is_built_with_memory_sanitizer():
         pytest.skip("Memory Sanitizer cannot work with third-party shared libraries")
 
-    err = instance.query_and_get_error("select catboostEvaluate(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11);")
-    assert "Illegal type UInt8 of first argument of function catboostEvaluate, expected a string" in err
+    err = instance.query_and_get_error(
+        "select catboostEvaluate(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11);"
+    )
+    assert (
+        "Illegal type UInt8 of first argument of function catboostEvaluate, expected a string"
+        in err
+    )
 
     instance.query("DROP TABLE IF EXISTS T;")
     instance.query("CREATE TABLE T(ID UInt32, A String) ENGINE MergeTree ORDER BY ID")
     instance.query("INSERT INTO T VALUES(0, 'test');")
-    err = instance.query_and_get_error("select catboostEvaluate(A, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11) FROM T;")
-    assert "First argument of function catboostEvaluate must be a constant string" in err
+    err = instance.query_and_get_error(
+        "select catboostEvaluate(A, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11) FROM T;"
+    )
+    assert (
+        "First argument of function catboostEvaluate must be a constant string" in err
+    )
     instance.query("DROP TABLE IF EXISTS T;")
 
 
 def testWrongNumberOfFeatureArguments(ch_cluster):
     if instance.is_built_with_memory_sanitizer():
         pytest.skip("Memory Sanitizer cannot work with third-party shared libraries")
 
-    err = instance.query_and_get_error("select catboostEvaluate('/etc/clickhouse-server/model/simple_model.bin');")
+    err = instance.query_and_get_error(
+        "select catboostEvaluate('/etc/clickhouse-server/model/simple_model.bin');"
+    )
     assert "Function catboostEvaluate expects at least 2 arguments" in err
 
-    err = instance.query_and_get_error("select catboostEvaluate('/etc/clickhouse-server/model/simple_model.bin', 1, 2);")
-    assert "Number of columns is different with number of features: columns size 2 float features size 2 + cat features size 9" in err
+    err = instance.query_and_get_error(
+        "select catboostEvaluate('/etc/clickhouse-server/model/simple_model.bin', 1, 2);"
+    )
+    assert (
```

Comment:
What an ugly formatting. But I'll better not argue with the autoformatter :smile: